src/App.jsx

```diff
@@ -29,6 +32,15 @@ export const App = () => {
       }
     ])
   };
+
+  // タスク削除用関数
+  const handleTodoDelete = (deleteId) => {
+    const newTodoItems = todoItems.filter((todoItem) => {
+      return todoItem.id !== deleteId.id;
```

Comment:
おそらくTodoItemで
```
const deleteId = {id};
// {id}は{ id: id }と同じ意味
```
のようにオブジェクトに変換して渡しているのでこのような書き方になっているのかと思います。
deleteIdはid(key)にpropsから受け取ったid(value)が入るオブジェクトとなっている。
オブジェクトではなくそのまま渡すことでスッキリ書くことができます。

```suggestion
      return todoItem.id !== deleteId;
```

Reply:
コミット「[75f83bc](https://github.com/hideki1516/react-todoapp/pull/3/commits/75f83bc438b45278fd036032d88227426012798a)」で修正しました。